FSW/src/util/memUtil.cpp

```diff
@@ -0,0 +1,250 @@
+/* memUtil.cpp handles NS2 payload memory allocation and assignment 
+ * Usage:
+ *  module functionality
+ *  function definitions
+ *  put function declarations in memUtil.hpp
+ * 
+ * Modules encompassed:
+ *  Data Processing
+ *  Science Memory Handling
+ *
+ * Additional files needed for compilation:
+ *  none
+ */
+
+/* - - - - - - Includes - - - - - - */
+// All libraries are put in memUtil.hpp
+// NS2 headers
+#include "../headers/memUtil.hpp"
+
+/* Module Variable Definitions */
+//TODO: should sampling rate and window length be configurable on flight?
```

Comment:
I think both of these should be configurable on flight. We may want to increase sampling rate if we determine that we have the ability to average more data. We may want to extend the buffer length to capture additional data to adjust calibration times.